README.md

```diff
@@ -1,43 +1,153 @@
-# Project 
-Exercise tracker
+<!-- PROJECT -->
+  <p align="center">
+   Phisical Fitness
+    <br />
+        <img src="/assets/images/github.png"
+        alt="GitHub"
+             height="80"></a>
+    <br />
+    ![Github](https://github.com/sisay2405/phiscal-fitness-app/blob/feature/readme/src/assets/images/github.png)
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+        <ul>
+        <li><a href="#built-with">Built With</a></li>
+        <li><a href="#wireframe">Wireframe</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
-## Description
+<!-- ABOUT THE PROJECT -->
+## About The Project
+This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum: exercise (add exercise, [pushup, pull, number, completed/not] save to storage , loop)
+* Create complex workouts easily
+* Define your own exercises with custom durations
+* Add , update and remove exercises 
 
+This app used Redux Toolkit and state slices to manage any state data needed across multiple components. In the case where data is created and used only within one component (or possibly one and a few of its direct children), consider using useState() or useReducer() and traditional prop drilling. Most apps will use a combination of "local" state management (useState() and useReducer()) and "global" state management (Redux or Context API).
 
-## Global requirements
-1. authentication/authorization
-2. manage exercises (crud)
+Apply the TypeScript static typing system. try to use TypeScript for the majority of React components! installed typescript, add support for TypeScript linting, and use either .ts or .tsx extension for the files to apply TypeScript type-checking on.
 
-### Import requirements
-1. exercise (add exercise, [pushup, pull, number, completed/not] save to storage , loop)
-2. exercise log
+Use the `BLANK_README.md` to get started.
 
-## highlight important things for next time
-try to make it detailed
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Screenshots
```

Comment:
This header is not showing up correctly styled for me. Not sure if it's the combination of HTML plus markdown?